build/android-build.py

```diff
@@ -1,120 +1,100 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
Why changed like this? I found python is installed in `usr/bin/python` in my machine.
```
A03101308207:~ minggo$ which python
/usr/bin/python
```